### PR TITLE
Feat/show trips

### PR DESCRIPTION
**Pull request by sebsch1337**
Please review my show trips page with some dummy data and TripCard component.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,22 +1,45 @@
...
       <Head>
-        <title>My App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
+        <title>Trips</title>
       </Head>
-
+      <TripsHeadline>
+        Start
+        <br />
```
Comment (philmetscher) [inline pages/index.js:16]: You should not use br (mentioned by the coaches in between)

File: components/TripCard.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <CountryName>{country}</CountryName>
+      </TextArea>
+    </CardBox>
+  );
+}
+
+const CardBox = styled.button`
+  width: 20em;
+  height: 20em;
+  filter: drop-shadow(0 0 1em #a1a1a1);
```
Comment (philmetscher) [inline components/TripCard.js:16]: u could use the color var in your GlobalStyle

**Review by philmetscher — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 1 above)
Comment (sebsch1337) [reply pages/index.js:16]: As i understand we shouldn't use it for spacing elements. In this case it's really used for its purpose; a line break for text.

File: components/TripCard.js (hunk 2 above)
Comment (sebsch1337) [reply components/TripCard.js]: Added a color var for drop-shadow. Will be useful in future tasks.

File: pages/index.js [hunk 3]
```diff
@@ -1,22 +1,45 @@
...
+import { dbTrips } from "../db";
 
 export default function Home() {
+  const [trips, setTrips] = useState(dbTrips);
   return (
     <div>
       <Head>
-        <title>My App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
```
Comment (philmetscher) [inline pages/index.js:10]: why did u remove those infos?

**Review by philmetscher — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 3 above)
Comment (sebsch1337) [reply pages/index.js:10]: I dont need page-specific meta tags. I'm gonna add them for the whole app in a later stage.

**Pull request merged**